search/tests/test_sorting.py

```diff
@@ -0,0 +1,117 @@
+"""
```

Comment:
I disagree: the meaning is actually to test that they do not change, because other part of the code can be impacted.
For example the default sorting is used in many places. 
The fact that the sorting is a tuple of 2 elements and the first is the index name is important.
The behavior of the functions needs to be consistent with what I defined here.
How do I test this stuff at high level?